src/components/dice-game/Dice.js

```diff
@@ -0,0 +1,338 @@
+                                setLoading(false)
+                            }}
+                            disabled={isLoading}
+                        />
+                        <label htmlFor="choicePair">Четные</label>
+                    </div>
+                </div>
+                {result !== null ? (
+                    resultImg()
+                ) : (
+                    <img
+                        className={
+                            isLoading
+                                ? `${styles.img} ${styles.isLoading}`
+                                : styles.img
+                        }
+                        src={DiceImg}
+                        title="dice"
+                    />
+                )}
+                {result !== null && (
+                    <h1 className={styles.result}>
+                        {isBetWin() ? 'ПОБЕДА !!!' : 'ПОРАЖЕНИЕ'}
+                    </h1>
+                )}
+                <div className={styles.buttonWrapper}>
+                    <button
+                        className={styles.button}
+                        onClick={placeBet}
+                        disabled={bet === null || isLoading}
+                    >
+                        Ставка
+                    </button>
+                    {token && (
+                        <button
+                            className={styles.button}
+                            onClick={showHistoryHandler}
+                        >
+                            История
+                        </button>
+                    )}
+                </div>
+                {showHistory && (
+                    <div className={styles.history}>
+                        <button onClick={() => setShowHistory(false)}>X</button>
+                        <h1>История</h1>
+                        <br />
+                        <ol className={styles.historyList}>
+                            {historyData.map(historyItem => {
+                                return (
+                                    <li
+                                        key={historyItem.id}
+                                        className={styles.historyListItem}
+                                    >
+                                        <p> Дата: {historyItem.dateTime}</p>
+                                        <p>
+                                            Ставка:{' '}
+                                            {getBetValue(historyItem.betType)} -
+                                            Результат:{' '}
+                                            {historyItem.droppedNumber} ПОБЕДА:{' '}
+                                            {historyItem.isBetWon ? '✅' : '❌'}
+                                        </p>
+                                        <hr />
+                                    </li>
+                                )
+                            })}
+                        </ol>{' '}
```

Comment:
`{' '}` why?

Reply:
gap
after the demo restyle the whole component